components/TutorialEndCard.js

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+export default function TutorialEndCard() {
+  return (
+    <>
```

Comment:
Don't need the fragment `<>`here because your `CardContainer` clutches the other html-elements.